src/pages/Profile/index.jsx

```diff
@@ -29,12 +37,23 @@ export default function Profile() {
       }
     };
   }
-  function isAvailableInTheList(list) {
-    return function (id) {
-      return list.find((game) => game.id === id);
-    };
+
+  function dragStartHandler(list, setList, data, ev) {
+    setDataTransfer({ from: [list, setList], data: { ...data } });
```

Comment:
this function can be further simplified, you don't need to store `list` and `setList` variable, all you need to do is store data of  card you are dragging and from its status you can know from which section you are dragging it, for example if its status is `want to play` then you know this card belongs to `want to play` list.

Reply:
then we need to add if conditions for three status (want to play, playing, played) in function?

Reply:
in this we just need to pass the list which needs to be updated

Reply:
that is ok to handle multiple condition, but that way it is  much more clear

Reply:
Fixed